prow/cluster/starter.yaml

```diff
@@ -94,6 +94,7 @@ spec:
         app: hook
     spec:
       terminationGracePeriodSeconds: 180
+      serviceAccountName: "hook"
```

Comment:
You'll need to add the service accounts to the `*_deployment.yaml` files as well.

Reply:
Since our deployments won't be using RBAC we can't actually add the service account since we won't define it. Maybe add it to the deployment files, but comment it out with an explanation?

Reply:
Addressed in latest commit

Reply:
It doesn't appear to be?
I still see non-commented lines like `serviceAccountName: "plank"` in the deployment files.

Reply:
discussed this offline, definitely comment out with an explanation, someday we will be on RBAC and need to uncomment :upside_down_face: (and also add these to the rules_k8s deployment)

Reply:
I added service accounts to the deployments and purposefully didnt comment them out, thought Ben's comment below was saying these wouldnt be deployed to prow.k8s.io.

Reply:
Latest commit contains commented out service accounts in deployment files